### ISSUE TITLE
System.ArgumentOutOfRangeException: 'Non-negative number required. Arg_ParamName_Name'

### ISSUE THREAD

**Issue by 0x410c** [MarkPflug/Sylvan.Data.Excel#26]
trace :
```
System.Text.UnicodeEncoding.GetString(byte[], int, int)
    Sylvan.Data.Excel.XlsbWorkbookReader.RecordReader.GetString(int, out int)
    Sylvan.Data.Excel.XlsbWorkbookReader.XlsbWorkbookReader(System.IO.Stream, Sylvan.Data.Excel.ExcelDataReaderOptions)
    Sylvan.Data.Excel.ExcelDataReader.Create(System.IO.Stream, Sylvan.Data.Excel.ExcelWorkbookType, Sylvan.Data.Excel.ExcelDataReaderOptions)
    Sylvan.Data.Excel.ExcelDataReader.Create(string, Sylvan.Data.Excel.ExcelDataReaderOptions)
```

please fix this

**Comment by MarkPflug** [comment]
Should be fixed in 0.1.8-b0002

**Comment by 0x410c** [comment]
this is kind of fixed in xlsb but i have a file where it occurs in xlsx, will create a seperate issue
